desktop/src/main/java/bisq/desktop/main/account/content/walletinfo/WalletInfoView.java

```diff
@@ -160,7 +160,7 @@ private void addXpubKeys(WalletService walletService, String currency, int gridR
 
     private void updateBalances(WalletService walletService) {
         if (walletService instanceof BtcWalletService) {
-            btcTextField.setText(btcFormatter.formatCoinWithCode(walletService.getBalance(ESTIMATED_SPENDABLE)));
+            btcTextField.setText(xmrFormatter.formatCoinWithCode(walletService.getBalance(ESTIMATED_SPENDABLE)));
```

Comment:
The btcFormatter should be used here and an additional conditional should be added for XmrWalletService which uses the XMR formatter, else throws an exception.

Reply:
you are right I didnt check all the occurrences and how is it used, my thinking was, that if btc was the base currency before and no other crypto currencies had their own Formatter, we just need to replace it everywhere for XMR. 

Maybe in this specific case BtcWalletService will need to be changed to XmrWalletService? I will have a look into that. 

But again, if there was a specific case for BTC in bisq, and not for any other crypto currencies I thought we should change it to XMR and not leave XMR and BTC. If there would be a special case for BTC, then why not for other crypto currencies?

Reply:
Eventually we should expect all instances of BtcWalletService and other Bitcoin-specific code to be fully replaced with XMR.  Until then, we are preserving their function and building XMR support in parallel.  When BtcWalletService and other Bitcoin-specific code are removed, we should be able to fully remove the BTC formatter.

Reply:
So basically we are getting there in steps and I want to avoid in-between steps with invalid states between Bitcoin and Monero.